AsyncDisplayKit/ASTableView.mm

```diff
@@ -75,6 +75,10 @@ - (void)setHighlighted:(BOOL)highlighted animated:(BOOL)animated
   _node.highlighted = highlighted;
 }
 
+- (BOOL)pointInside:(CGPoint)point withEvent:(UIEvent *)event {
```

Comment:
Great idea!  Should we bridge hitTest too?  Will that work correctly with the logic already in ASDisplayNode that is used for hitTesting, if we bridge it directly to the node without a special implementation? (it should)

Bump the { on the next line to match the style, and I'm happy to merge this.  Optionally, add a comment to the header indicating that this is supported.